### PR TITLE
Tweaks to documentation

### PR DESCRIPTION
**Pull request by bwspenc**
ref #12
Rename InputStructure.md file to avoid name conflict
Improved determination of MOOSE_DIR

### CODE REVIEW

File: doc/moosedocs.py [hunk 1]
```diff
@@ -3,11 +3,13 @@
 import os
 
 # Locate MOOSE directory
-MOOSE_DIR = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '../moose'))
+os.chdir(os.path.abspath(os.path.dirname(__file__)))
+MOOSE_DIR = os.getenv('MOOSE_DIR', os.path.join(os.getcwd(), '..', 'moose'))
 if not os.path.exists(os.path.join(MOOSE_DIR, 'libmesh')):
-  MOOSE_DIR = os.path.join(os.getenv('HOME'), 'projects', 'moose')
+  MOOSE_DIR = os.path.join(os.getcwd(), '..', '..', 'moose')
```
Comment (sapitts) [inline doc/moosedocs.py]: What's this line do? make sure that the moose dir can be found no matter what the top-level directory is named?

Comment (bwspenc) [reply doc/moosedocs.py]: Line 7 first looks for moose as a submodule (blackbear/moose).  If that doesn't exist, this line (9) tries the directory alongside blackbear: (blackbear/../moose).  Previously, it looked in ~/projects/moose, which is making an assumption that is different from how MOOSE is found for building the code.

Comment (bwspenc) [reply doc/moosedocs.py]: We should make the same change to BISON.

Comment (sapitts) [reply doc/moosedocs.py]: will do.  thanks for the explanation

**Pull request merged**